qlib/contrib/model/double_ensemble.py

```diff
@@ -0,0 +1,247 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import lightgbm as lgb
+import numpy as np
+import pandas as pd
+
+from ...model.base import Model
+from ...data.dataset import DatasetH
+from ...data.dataset.handler import DataHandlerLP
+from ...log import get_module_logger
+
+
+class DEnsembleModel(Model):
+    """Double Ensemble Model"""
+
+    def __init__(
+        self,
+        base="gbm",
+        loss="mse",
+        k=6,
+        enable_sr=True,
+        enable_fs=True,
+        alpha1=1.0,
+        alpha2=1.0,
+        bins_sr=10,
+        bins_fs=5,
+        decay=None,
+        sample_ratios=None,
+        sub_weights=None,
+        epochs=100,
+        **kwargs
+    ):
+        self.base = base  # "gbm" or "mlp", specifically, we use lgbm for "gbm"
+        self.k = k

```

Comment:
Please give more details about the parameters.  
Give it a full name at least

Reply:
OK. The new names are "base_model" and "num_models".